Zend/zend_builtin_functions.stub.php

```diff
@@ -68,24 +77,40 @@ function function_exists(string $function): bool {}
 
 function class_alias(string $class, string $alias, bool $autoload = true): bool {}
 
+/**
+ * @return array<int, string>
+ * @refcount 1
+ */
 function get_included_files(): array {}
 
-/** @alias get_included_files */
+/**
+ * @return array<int, string>
+ * @refcount 1
+ * @alias get_included_files
+ */
 function get_required_files(): array {}
 
 function trigger_error(string $message, int $error_level = E_USER_NOTICE): bool {}
 
 /** @alias trigger_error */
 function user_error(string $message, int $error_level = E_USER_NOTICE): bool {}
 
-/** @return string|array|object|null */
+/**
+ * @return string|array<int, string|object>|object|null
+ * @refcount N
```

Comment:
Similar to the `@refcount 0` case, as `@refcount N` is the default state for non-scalar types, can we omit it and generate the func info based on presence of more specific return type?

Reply:
Can these `@refcount N` lines be dropped after your last change?

Reply:
Yeah, I'll do that, thanks for pointing them out! Furthermore, I could forbid the addition of `@refcount 1` and `@refcount N`. Another question I had: what to do with functions with `void` and `never` return type? Can I disallow adding a refcount (both implicitly and explicitly) to them?

I'll rebase to master soon.